frontend/app/components/footer.tsx

```diff
@@ -6,7 +6,7 @@ export function Footer() {
   const { t } = useTranslation(['common']);
 
   return (
-    <footer id="footer" className="bg-[#26374A]">
+    <footer id="wb-info" tabIndex={-1} className="bg-[#26374A]">
```

Comment:
![image](https://github.com/user-attachments/assets/13d9222b-16bc-443c-a907-34756d06009c)

```suggestion
    <footer id="wb-info" tabIndex={-1} className="bg-slate-700">
```

Reply:
Changes applied. This also now matches the primary button bgcolor `slate-700`.